keep/alertmanager/alertmanager.py

```diff
@@ -110,6 +110,7 @@ def _run_alerts(self, alerts: typing.List[Alert]):
                 self.logger.error(
                     f"Error running alert {alert.alert_id}", extra={"exception": e}
                 )
+                raise
```

Comment:
the current behavior does not make sense (if you have multiple alert it silently success, e.g. you had 10 alerts and the first 9 failed due to syntax error, and the last one is ok, the overall will be ok)

Reply:
sounds like a breaking change -- what are the effects?

Reply:
when you run a single alert - it will raise an error if the alert (which IMO should be the behavior) and you run multiple alerts (e.g. directory) it will fail if one of the alerts fails